src/oci-umount.c

```diff
@@ -1019,17 +1010,17 @@ static int parseBundle(const char *id, yajl_val *node_ptr, char **rootfs, struct
 	}
 
 	/* Read the entire config file */
-	snprintf(errbuf, BUFLEN, "failed to read config data from %s", config_file_name);
-	configData = getJSONstring(fp, (size_t)CHUNKSIZE, errbuf);
+    snprintf(err.text, sizeof(err.text), "failed to read config data from %s", config_file_name);
+    configData = getJSONstring(fp, (size_t)CHUNKSIZE, err.text);
```

Comment:
All this is part of generic cleanup and has nothing to do with json conversion? Can we break the patch down in to a patch series and all the changes which are cleanup and not related to json converstion, move them into a separate patch. That way reviewing becomes easier.